Themes/default/languages/index.english.php

```diff
@@ -76,7 +76,7 @@
 $txt['remove'] = 'Remove';
 $txt['start_new_topic'] = 'Start new topic';
 
-$txt['login'] = 'Login';
+$txt['login'] = 'Sign in';
```

Comment:
i personal run always in the issue that i can't distinguish "sign in" and "sign up" dunno if this a personal issue.

Reply:
Do you mean they're too similar visually, or that you mix up their meaning, or something else?

Reply:
"Sign up" and "Sign in" are both looks very similiar and for me is sign up and sign in same meaning, 
only when they stand by side i got a good chance to remeber which is the right one.

Reply:
A bit of both, then. For me, it always helps that "sign in" / "log in" is similar (and maybe also "check in", like in a hotel where you already "registered" beforehand). And "sign (me) up" ("I wasn't on board before, but I want to be—where do I put my signature?") is kind of a level-up from being an unregistered guest. :wink:

Reply:
Poking around to some websites, it seems "Sign in" is becoming more used.  Even Youtube and GitHub are using "Sign In", but if you look at the url, they usually mention "login".  So it seems to me that they are interchangeable.  I would say the preference based on the url and text comparison to see if they use signin/login in either.  Some are using the same, some are using them one or the other.  Although I didn't find one that used both signup in the url and text.

Sites I looked around at: Reddit, Yahoo, Github, Outlook.com, Youtube, Google, Bitbucket, Netflix, Amazon, Linode, Digitalocean, Apple (ID), Pandora, Spotify and Discord.

Reply:
I don't honestly care whether we use "sign up", "sign in", and "sign out", or "register", "login", and "logout", but we should consistently use one set of terms or the other.

Reply:
I've just spent a few minutes looking at that list of sites @jdarwood007 mentioned above, plus Twitter and Facebook, and overall it does seem like a pretty mixed bag.

However, I did notice that most of those sites avoid placing "Sign In" and "Sign Up" buttons right next to each other: 
- The general trend seems to be that if "Sign Up" is used, then "Log In" or some other phrase is used, or conversely that if "Sign In" is used, then "Create Account" or some alternative is used.
-  Google, YouTube, and Yahoo take a different approach by presenting a single "Sign In" button and then relying on a "Create Account" button on their login form to catch and redirect people who don't have an account yet.
- Overall, it seems like the "Sign Up" + "Log In" combination tends to be preferred by sites that appeal to more tech savvy demographics, whereas the "Sign In" + "Create Account" combination tends to be preferred by sites that aim for broader appeal.

Based on these findings, I now suspect that @albertlast's concerns are well founded. In theory, "sign in" and "sign up" are both more user friendly than the old fashioned "login" and "register" terminology. Yet it seems that these companies, who spend ridiculous amounts of money researching such things, have almost all come to the conclusion that using both "sign in" and "sign up" at the same time is a bad idea. It seems quite plausible to me that user confusion, like @albertlast describes, is the reason.

Therefore, I now agree with going back to "Log In" and "Log Out", while keep "Sign Up". If all other factors were equal, I would probably have preferred "Sign In", "Sign Out", and "Create Account", but our existing URL query parameters already match the first option. I will submit the changes shortly.

Reply:
I did, however, change "Login" and "Logout" to "Log in" and "Log out". The words without spaces are nouns, naming processes or functions, whereas the words with spaces are verbs, and the latter is what we use those $txt strings for.

Reply:
Like i said in the start, when i'm the only person, than it's fine,
but i got the issue on some pages...

Reply:
Don't worry about it, @albertlast. I'm already convinced that you were right, and I am glad you mentioned it. 🙂